controllers/nginx/Makefile

```diff
@@ -30,7 +30,7 @@ ALL_ARCH = amd64 arm arm64 ppc64le
 
 QEMUVERSION=v2.9.1
 
-IMGNAME = nginx-ingress-controller
+IMGNAME = nginx-ingress-controller-fork
```

Comment:
I will change this once we are okay with the PR and remove the ingress from the other repo